src/cryptonote_core/service_node_list.h

```diff
@@ -176,11 +176,11 @@ namespace service_nodes
         bool unreachable_for(std::chrono::seconds threshold, const std::chrono::steady_clock::time_point& now = std::chrono::steady_clock::now()) const;
 
     };
-    reachable_stats ss_reachable;
-    reachable_stats lokinet_reachable;
+    reachable_stats ss_reachable{};
+    reachable_stats lokinet_reachable{};
 
     // Unlike all of the above (except for timestamp), these values *do* get serialized
-    std::unique_ptr<uptime_proof::Proof> proof;
+    std::unique_ptr<uptime_proof::Proof> proof{};
```

Comment:
This one doesn't need it because std::unique_ptr has a default constructor.  (It doesn't hurt, it's just already what has to happen for types with a constructor).